tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Tags.cpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+
+namespace {
+struct ArbitraryFrame;
+}  // namespace
+
+#define _TEST_SIMPLE_TAGS(D, F)                                            \
```

Comment:
I’d prefer to make this a function template, instead of a macro. Or did you make this a macro for a particular reason?

Reply:
changed